using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md

```diff
@@ -1,56 +1,58 @@
 # Campaign Classic API reference
 
-## Version of the Campaign Classic extension
+## extensionVersion
 
-The `extensionVersion()` API returns the version of the Campaign Classic extension that is registered with the Mobile Core extension.
-
-To get the version of the Campaign Classic extension, use the following code sample:
+The `extensionVersion` API returns the version of the Campaign Classic extension that is registered with the Mobile Core extension.
 
 {% tabs %}
 {% tab title="Android" %}
-#### Java
+### Java
+
+**Example**
 
 ```java
 String campaignClassicExtensionVersion = CampaignClassic.extensionVersion();
 ```
 {% endtab %}
 
 {% tab title="iOS" %}
-**Objective C**
+### Objective-C
+
+**Example**
 
 ```objectivec
 NSString *campaignClassicExtensionVersion = [ACPCampaignClassic extensionVersion];
 ```
 
-**Swift**
+### Swift
+
+**Example**
```

Comment:
One nice thing to do would be to reorder Swift first, then Objective-C across all examples. We can probably do that in a separate PR if you want.